### ISSUE TITLE
[Browse lists] Increase contrast for links on the browse list page

### ISSUE THREAD

**Issue by rladdusaw** [pulibrary/orangelight#4839]
### Steps to recreate
Go to a browse list. The links on a gray background has insufficient contrast.

### URL
https://catalog.princeton.edu/browse/call_numbers?start=14593860

### Issues
color-contrast

### Severity
Serious

### WCAG Conformance level (A, AA, AAA)
AA

### User impact
Text with insufficient contrast is difficult to read

### Recommended fix
Increase contrast

### Acceptance criteria
- [ ] I can successfully use this feature with Voice Over in Safari.
- [ ] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave
